docs/didkit/install.md

```diff
@@ -11,15 +11,38 @@ title: Installation
 
 
 
-DIDKit can be installed manually or in containerized form.
+DIDKit can be installed as a package via [cargo](#cargo-install),
+[manually](#manual) from source, or in [containerized](#container) form.
+
+## Cargo install
+
+1. To install the DIDKit command line program on GNU/Linux, MacOS, or
+   Windows+WSL, first install
+   [cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html). 
+2. Install `build-essential` or equivalent tools if they aren't already installed.
+3. Then simply run `cargo install` for the given target package: 
+`didkit-cli | didkit-http`.  
+For example, for DIDKit CLI, run:
+
+```
+cargo install didkit-cli
+```
+
+This will add the binary didkit to your Cargo installation (typically
+~/.cargo/bin), which can be added to your system's PATH for ease of use.
+
 
 ## Manual
 
 DIDKit is written in [Rust][]. To get Rust, you can use [Rustup][].
 
-We do not depend on any Rust nightly features, so our installation instructions assume `stable` versions; be sure to [switch the installation defaults][installing-rust] to `nightly` if the calling application depends on them.
+We do not depend on any Rust nightly features, so our installation instructions
+assume `stable` versions; be sure to [switch the installation
```

Comment:
Maybe add "recent stable version". At the moment it's 1.51 but it will likely go up soon as we transition to the 2021 edition.

Reply:
added